config/jobs/kubernetes/test-infra/test-infra-periodics.yaml

```diff
@@ -88,7 +90,97 @@ periodics:
         value: gsutil
       - name: CLOUD_BUCKET
         value: gs://prow-cifuzz-test/
+      - name: CFL_PLATFORM
+        value: prow
   annotations:
     testgrid-dashboards: sig-testing-misc
     testgrid-tab-name: test-infra-fuzz
     description: Runs clusterfuzzlite every hour
+
+- name: cfl-coverage-report
```

Comment:
If these are repo specific please name them accordingly. Job names are unfortunately not namespaced.